src/dependenpy/cli.py

```diff
@@ -185,7 +185,7 @@ def _run(opts, dsm):
             dsm.print_graph(format=opts.format, output=output, depth=depth, indent=indent)
 
 
-def main(args: Optional[List[str]] = None) -> int:  # noqa: WPS231
+def main(args: List[str] | None = None) -> int:  # noqa: WPS231
```

Comment:
You can also use `list[str]` instead of importing `List` form `typing` now that you added future annotations, though that's not mandatory (I'm fine with typing classes if you prefer it).